### ISSUE TITLE
Epub documentation fails to render, throws errors

### ISSUE THREAD

**Issue by kevzettler** [godotengine/godot-docs#7498]
**Your Godot version:**
Tried to download epub documentation from: https://nightly.link/godotengine/godot-docs/workflows/build_offline_docs/master/godot-docs-epub-master.zip

**Issue description:**
Every page of the epub renders an error:

```
“This page contains the following errors:error on line 45 at column 10: Opening and ending tag mismatch: meta line 0 and head
Below is a rendering of the page up to the first error.”

Excerpt From
Godot Engine
Juan Linietsky, Ariel Manzur and the Godot community
This material may be protected by copyright.
```

Fails to render on Apple Books and Kindle

![image](https://github.com/godotengine/godot-docs/assets/126209/df5f188a-0a23-498a-961e-32601176deff)

**Comment by eimfach** [comment]
I would like to look into the issue and fix it, where should I start ? Is it just the meta and head tags ? Looks like a small issue on the surface ..

**Comment by skyace65** [comment]
This was fixed by #8207.
